### PR TITLE
Allow comments for fields of custom type constructor

### PR DESCRIPTION
**Pull request by tomwhatmore**
Addresses #568

### CODE REVIEW

File: src/erl.rs [hunk 1]
```diff
@@ -103,7 +103,7 @@ pub fn records(module: &TypedModule) -> Vec<(&str, String)> {
         .filter(|constructor| !constructor.args.is_empty())
         .flat_map(|constructor| {
             let mut fields = Vec::with_capacity(constructor.args.len());
-            for (label, _) in constructor.args.iter() {
+            for (label, ..) in constructor.args.iter() {
```
Comment (lpil) [inline src/erl.rs:106]: TIL

**Comment by lpil** [comment]
Would you mind updating the changelog too? Thanks

**Comment by tomwhatmore** [comment]
Updated

**Comment by lpil** [comment]
✨

**Pull request merged**